frontend/src/grpcweb/middlewares.ts

```diff
@@ -0,0 +1,76 @@
+import { ClientError, ServerError, Status } from "nice-grpc-common";
+
+import { pushNotification, useAuthStore } from "@/store";
+import { router } from "@/router";
+import { t } from "@/plugins/i18n";
+import { ClientMiddleware } from "nice-grpc-web";
+
+export type SilentRequestOptions = {
+  /**
+   * if set to true, will NOT show push notifications when request error occurs.
+   */
+  silent?: boolean;
+};
+
+export const errorNotificationMiddleware: ClientMiddleware<SilentRequestOptions> =
```

Comment:
Can we add a comment with reference docs right here?

Reply:
Sure thing